src/Symfony/Bundle/FrameworkBundle/Tests/Console/ApplicationTest.php

```diff
@@ -226,6 +229,34 @@ public function testRunOnlyWarnsOnUnregistrableCommandAtTheEnd()
         $this->assertContains(trim('[WARNING] Some commands could not be registered:'), trim($display[1]));
     }
 
+    public function testSuggestingPackagesWithExactMatch()
+    {
+        $result = $this->createEventForSuggestingPackages('server:dump', []);
+        $this->assertRegExp('/You may be looking for a command provided by the/', $result);
+    }
+
+    public function testSuggestingPackagesWithPartialMatchAndNoAlternatives()
+    {
+        $result = $this->createEventForSuggestingPackages('server', []);
+        $this->assertRegExp('/You may be looking for a command provided by the/', $result);
+    }
+
+    public function testSuggestingPackagesWithPartialMatchAndAlternatives()
+    {
+        $result = $this->createEventForSuggestingPackages('server', ['server:run']);
+        $this->assertNotRegExp('/You may be looking for a command provided by the/', $result);
```

Comment:
`the` should be removed I guess

Reply:
Done. I was fixing it when you commented